enterprise/formatpainter.md

```diff
@@ -23,7 +23,7 @@ To try out Format Painter and Tiny Cloud-delivered editor, the first step is to
 
 ### Buy TinyMCE Format Painter
 
-Start with the [pricing page](https://www.tiny.cloud/pricing/)<!--this [dedicated product page](https://about.tiny.cloud/products/formatpainter/)--> to see all the flexible pricing options. OEM and enterprise customers should [contact sales directly](https://www.tiny.cloud/contact/).
+Start with this [dedicated product page](https://apps.tiny.cloud/products/format-painter/) to see all the flexible pricing options. OEM and enterprise customers should [contact sales directly](https://www.tiny.cloud/contact/).
```

Comment:
https://apps.tiny.cloud/products/format-painter/ page does not exist

Reply:
Product links will be available post GA.